layer/CMakeLists.txt

```diff
@@ -52,7 +52,8 @@ target_sources(VkLayer_gfxreconstruct
                    ${CMAKE_CURRENT_LIST_DIR}/trace_layer.cpp
                    ${CMAKE_SOURCE_DIR}/framework/encode/custom_layer_func_table.h
                    ${CMAKE_SOURCE_DIR}/framework/generated/generated_layer_func_table.h
-                   $<$<BOOL:WIN32>:${CMAKE_CURRENT_LIST_DIR}/trace_layer.def>
+                   $<$<BOOL:WIN32>:${CMAKE_CURRENT_LIST_DIR}/trace_layer.def
+                   ${CMAKE_SOURCE_DIR}/version.rc>
```

Comment:
Does this work for other platforms? Should it be included on Windows only?
`$<$<BOOL:WIN32>:${CMAKE_SOURCE_DIR}/version.rc`

Reply:
This `.rc` file is a Windows thing so yeah I could add that bool check too.

Reply:
Done